docs/manual/index.html

```diff
@@ -362,12 +363,9 @@ <h2> Contents </h2>
   <section id="pytket-user-manual">
 <h1>Pytket User Manual<a class="headerlink" href="#pytket-user-manual" title="Permalink to this heading">#</a></h1>
 <p>This manual refers to the following versions of pytket and extensions:</p>
-<div class="highlight-text notranslate"><div class="highlight"><pre><span></span>pytket[ZX] == 1.17.1
-pytket-projectq == 0.28.0
-pytket-qiskit == 0.41.0
-pytket-qsharp == 0.34.0
+<div class="highlight-text notranslate"><div class="highlight"><pre><span></span>pytket[ZX] == 1.18.0
+pytket-qiskit == 0.42.0
```

Comment:
Have you updated this in the requirements file as well?

Reply:
It looks like 0.42.0 was already the version listed in the requirements, I am not sure why this is shown as a change here? Do you understand this?

Reply:
ah I have separated the requirements for the manual and examples into separate txt files in https://github.com/CQCL/pytket/pull/248/files This is the first time I have rebuilt the manual since the chage.

Should I change the pytket-qiskit version back to 0.41.0 for the manual?

Reply:
Ah okay, no need to set that back.
In the past we had this automatically updated, but it looks like that is not longer the case.